### PR TITLE
Added Kruskal's Algorithm 

### PR DESCRIPTION
**Pull request by W-ight**
## Related Issue

Fixed: #561

## Description

Added the Kruskal's Minimum Spanning Tree Algorithm in DSA Algorithm Section

## Type of PR

- [x] Feature enhancement
- [x] Documentation

## Checklist
- [x] I have performed a self-review of my code.
- [x] I have read and followed the Contribution Guidelines.
- [x] I have tested the changes thoroughly before submitting this pull request.
- [x] I have provided relevant issue numbers.
- [x] I have commented my code, particularly in hard-to-understand areas.

### CODE REVIEW

File: dsa/Algorithms/kruskal's.md [hunk 1]
```diff
@@ -0,0 +1,345 @@
...
+0 0 0 1 1 3 6 4
+1 3 6 2 4 4 4 5
+2 3 4 3 2 5 6 7
+Kruskals path is: 
+0 1 0 1 0 4 
+1 4 3 2 6 5 
+
+```
+### Time and Space Complexity:
+* Time complexity: The time complexity of kruskal's algorithm depends on two operations i.e. sorting of all edges and union-find operation.Thus, the overall time complexity is: **O(ElogV)**
```
Comment (Ajay-Dhangar) [inline dsa/Algorithms/kruskal's.md:344]: replace `**O(ElogV)**` to `$O(ElogV)$`

File: dsa/Algorithms/kruskal's.md [hunk 2]
```diff
@@ -0,0 +1,345 @@
...
+* Space complexity: The space complexity of kruskal's algorithm includes storage for edges, union-find data structure and storage for the MST. Thus the overall space complexity is: **O(E+V)**
```
Comment (Ajay-Dhangar) [inline dsa/Algorithms/kruskal's.md:345]: replace `**O(E+V)**` to `$O(E+V)$`

**Review by Ajay-Dhangar — CHANGES_REQUESTED** [review]
update your code based on suggestions

File: dsa/Algorithms/kruskal's.md (hunk 2 above)
Comment (W-ight) [reply dsa/Algorithms/kruskal's.md:345]: alright

**Review by Ajay-Dhangar — APPROVED** [review]
Done

**Pull request merged**